### ISSUE TITLE
Translations cleanup

### ISSUE THREAD

**Issue by ddohler** [azavea/echo-locator#532]
We've added new text as part of the Phase II work. We'll need to send those strings to BHA for translation. This task is to assist with gathering the strings together (e.g. as a spreadsheet) so that we can send them to BHA, and to insert the new translations once received.

**Comment by ktohalloran** [comment]
@ddohler what should we do about the language on the login page and magic link email? Are these phrases something we should get translations for now for in case we get to set up later, or should this list be limited to the phrases on pages inside the app? In a similar vein, does this card include setting up translation where it's needed (for example, the "logout" phrase in the header is currently not connected to `react18next` and would need to be connected and translated)?
EDIT: I realized that if you set the language in the app and then log out, that language preference is carried out onto the login page, at least until refresh. So it may be worth getting those translations just in case that happens, though that does seem to be a very specific case.

**Comment by ddohler** [comment]
@ktohalloran Good questions! The scope of this task is to make sure that any strings that appear on a screen that has a language switcher are translated accurately if the user switches the language on that screen. So that includes:
1. Strings that are already translated but that we've changed on those screens (like the "voucher" -> "Boston Housing Authority voucher" case).
2. Strings that we've added on those screens (like "Logout")

But keeping the budget in mind, I would avoid translations on any screens that don't display a language switcher. So that includes the login screen and the sign-in emails; we don't need to translate those until/unless we add a language switcher to the login screen.
